carl_interactive_manipulation/include/carl_interactive_manipulation/create_parking_spots.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef CREATE_PARKING_SPOTS_H_
+#define CREATE_PARKING_SPOTS_H_
+
+#include <ros/ros.h>
+#include <interactive_markers/interactive_marker_server.h>
+#include <visualization_msgs/Marker.h>
```

Comment:
unsued, but interactive markers are used (happens to work since interactive_marker_server imports them)